apps/mobile/src/components/add-wallet/add-wallet-sheet.layout.tsx

```diff
@@ -77,7 +79,14 @@ export function AddWalletSheetLayout({
       }}
     >
       <AnimatedBox style={animatedStyle}>
-        <Box width="100%" style={{ height: 200, overflow: 'hidden' }} bg="ink.text-primary">
+        <Box
+          width="100%"
+          style={{ height: 200, overflow: 'hidden' }}
+          bg={whenTheme({
```

Comment:
I would love it for our themes to be much less `theme` aware - eg. the tokens adapt better to the theme, `ink.background-secondary` should be all the `bg` needs and the theme adapts more elegantly.

Reply:
cc @kyranjamie / @tigranpetrossian

Reply:
`ink.background-secondary` is a semantic token, is restyle not set up to handle it conditionally?